### DISCUSSION TITLE
When do you expect the pycord-switch branch merges?

### DISCUSSION THREAD

**Discussion by xferah** [x3onkait/AEXOL#6] [Q&A] [answered]
Currently the running bot is the legacy version before branch and it's expected that users to keep using that version until the merge event. When we have to wait?

**Comment by KnightChaser** [comment, marked as answer]
I'm currently working on branches every day, but it expects a half month if I work every day. Also, some features will be added or modified at the time of the merge. Please remind the base package `pycord` what I'm transitioning has some different parts of `discord.py` even though they are compatible with each other, so unavailable legacy features can be deleted.

**Reply by xferah** [reply]
OK, I got it. thx
